tests/test_channel.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2023 Google LLC.
+# Copyright (c) Microsoft Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from test_helpers import execute_command
+
+
+# TODO(#294): Implement E2E tests.
+@pytest.mark.asyncio
+async def test_channel(websocket, context_id):
+    # The channel was successfully created if there's no thrown exception.
+    await execute_command(
+        websocket,
+        {
+            "method": "script.callFunction",
+            "params": {
+                # A small delay is needed to avoid a race condition.
+                "functionDeclaration": "(binding) => setTimeout(() => binding('MY_MESSAGE'), 1)",
```

Comment:
This tests the serialization, but we have to test message as well.

Reply:
TODO: subscribe to event (e.g. similar to network)

TODO: read next message, use read JSON helper